main/getting-started/deploying.md

```diff
@@ -1,63 +1,94 @@
 # Deploying Smart Contracts
 
-The deploy command in the [agoric command line tool](https://agoric.com/documentation/getting-started/agoric-cli-guide.html#agoric-deploy) 
+The `agoric deploy` command in the [agoric command line tool](https://agoric.com/documentation/getting-started/agoric-cli-guide.html#agoric-deploy) 
 supports deploying contracts and off-chain web applications that talk to them. The command 
 has two primary uses:
 
 * Deploy smart contract source code onto the blockchain
-* Deploy and setup an application program to the local server
+* Deploy and setup an application program to a local server running an Agoric process
 
-The deployment process uses the `agoric deploy` command to run your Dapp's `contract/deploy.js` and `api/deploy.js` scripts. You can use the deploy scripts created when you copied an existing Dapp into your directory as they are, or you can modify the scripts as suggested later in this document.
+Use the `agoric deploy` command to run your Dapp's `contract/deploy.js` 
+and `api/deploy.js` scripts. You can use the deploy scripts created when you copied an existing 
+Dapp into your directory as they are, or you can modify the scripts as suggested later in this document.
+
+Remember, your Dapp has three primary subdirectories:
+- `contract/`which contains files relating to your smart contract itself.
+- `api/`which contains files enabling the UI frontend to communicate via HTTP/WebSocket to an on-chain backend contract instance and start your Dapp contract instance and backend.
+- `ui/` which contains files relating to your contract's user interface.
+
+## How it works
+
+All deployment happens via the local running Agoric process. This is usually the `ag-solo` process, 
```

Comment:
My only concern about "the local agoric process" is that it might not be local. Since it almost always is, I think that's probably an ok simplification.